alb/alb.go

```diff
@@ -124,6 +124,13 @@ func (a *alb) Health() error {
 	return fmt.Errorf("have not attached to all frontends %v yet", a.targetGroupNames)
 }
 
+func (a *alb) Readiness() error {
+	if !a.readyForHealthCheck.Get() {
```

Comment:
If I understand correctly, the difference in the way that the `Health` and `Readiness` functions are implemented could mean that Feed reports as ready but not healthy. Couldn't that cause some problems?

Reply:
1. I don't think it would cause problems
2. I don't even think it could happen.  If readyForHealthcheck == false, then it will report healthy but not ready.  If readyForHealthcheck == true, then ready will report exactly what healthy reports/